_translations/en.yml

```diff
@@ -285,23 +285,23 @@ en:
-    walletmyceliumwallet: "Mycelium Bitcoin Wallet for Android is designed for security, speed, and ease of use. It has unique features to manage your keys and for cold storage and offers compatibility with Trezor and others."
-    walletcoin: "Coin is a free online bitcoin wallet, which you can use to make worldwide payments for free. It makes paying with bitcoins easy and secure available anywhere on your phone or desktop."
+    walletbrd: "BRD is designed to be easy to use for new Bitcoin users. It's also secure. BRD connects directly to the Bitcoin network, and uses your device's built-in hardware encryption to keep your money safe."
+    walletmyceliumwallet: "Mycelium Bitcoin Wallet is designed for security, speed, and ease of use. It also has unique features to manage your keys alongside cold storage and hardware wallets."
+    walletcoin: "Coin is a free online bitcoin wallet. It makes paying with bitcoins easy and secure available anywhere on your phone or desktop."
     walletbitgo: "BitGo is a high-security multi-sig wallet, which protects your bitcoin from theft and loss. You maintain full custody; BitGo cannot spend or freeze funds. BitGo wallets are easy to use and offer advanced security features such as spending limits and multi-user access."
-    wallettrezor: "Trezor is a hardware wallet providing a high level of security without sacrificing convenience. Unlike cold storage, Trezor is able to sign transactions while connected to an online device. That means spending bitcoins is secure even when using a compromised computer."
-    walletcoldcard: "Coldcard is an easy to use, ultra-secure, open-source and affordable hardware wallet that is easy to back up via an encrypted microSD card. Your private key is stored in a dedicated security chip. MicroPython software design allows you to make changes."
-    walletkeepkey: "KeepKey is a hardware wallet that makes bitcoin security simple. When you entrust KeepKey with your money, every bitcoin transaction you make must be reviewed and approved via its OLED display and confirmation button."
+    wallettrezor: "Trezor is a hardware wallet providing a high level of security without sacrificing convenience. Unlike cold storage, Trezor is able to sign transactions while connected to an online device."
+    walletcoldcard: "Coldcard is an easy to use, secure, open-source and affordable hardware wallet that is easy to back up via an encrypted microSD card. Your private key is stored in a dedicated security chip. MicroPython software design allows you to make changes."
+    walletkeepkey: "KeepKey is a hardware wallet that makes bitcoin security more simple. When you entrust KeepKey with your money, every bitcoin transaction you make must be reviewed and approved via its OLED display and confirmation button."
```

Comment:
I suggest leaving out "more".  We call other wallets simple, and easy, and secure (without the more).

Reply:
8db058eb